Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_Malware_-_Incident_Enrichment.yml

```diff
@@ -1215,7 +1214,7 @@ tasks:
       append:
         simple: 'true'
       key:
-        simple: MicrosoftATP.Alert.Evidence(val.entityType.length>0).device_name
```

Comment:
Instead of removing the DT - try to change the expression to `=== 0` and not `>0`

Reply:
if works - remove the task you added to verify the date

Reply:
done

Reply:
This DT expression also lead to the same error:
![Screenshot 2023-09-26 at 16 42 38](https://github.com/demisto/content/assets/112805149/a064c0a9-3bc4-4695-a57b-49b6c149febb)